bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/TapoControlHandlerFactory.java

```diff
@@ -42,15 +46,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
 /**
- * The {@link ThingHandler} is responsible for handling commands, which are
+ * The {@link TapoControlHandler} is responsible for handling commands, which are
  * sent to one of the channels.
  *
  * @author Christian Wild - Initial contribution
  */
 @Component(service = ThingHandlerFactory.class, configurationPid = "binding.tapocontrol")
 @NonNullByDefault
 public class TapoControlHandlerFactory extends BaseThingHandlerFactory {
+    public static final Gson GSON = new GsonBuilder().disableHtmlEscaping().excludeFieldsWithoutExposeAnnotation()
+            .create();
```

Comment:
Using `.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)` you can save a lot of annotations.

Reply:
Maybe i save some annotiations with underlines but i had to add others witch camelCase because it tapo used mixed FieldNamePolicies.